src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java

```diff
@@ -101,7 +95,7 @@ public static <T> T execute(final RequestModel<T> requestModel) throws MandrillA
 						.setConnectionManager(connexionManager).useSystemProperties()
 						.build();
 			}
-            log.debug("starting request '" +requestModel.getUrl()+ "'");
+            log.debug("starting request '{}'", requestModel.getUrl());
```

Comment:
Switched to new style of logging using {} instead of concatenation. This by itself can prevent some type of attacks and performance bottlenecks.